.github/workflows/ci-build.yaml

```diff
@@ -116,6 +117,7 @@ jobs:
           - 8.0.14
           - 8.1.0
           - 8.1.1
+          - 8.2.0RC2
```

Comment:
Failure is down to a deprecation notice due to the new dynamic property prohibition; the choices are: wait for upstream fix in https://github.com/ezimuel/ringphp - or disable deprecated notices for this test.

```
TEST 20/31 [tests/017-elastic-7-support.phpt]
========DIFF========
     Elasticsearch\Client->index
     Elasticsearch\Client->get
     Elasticsearch\Client->search
005+ 
006+ Deprecated: Creation of dynamic property GuzzleHttp\Ring\Client\CurlMultiHandler::$_mh is deprecated in /tmp/scout_elastic_test/vendor/ezimuel/ringphp/src/Client/CurlMultiHandler.php on line 47
     array(%d) {
       [%d]=>
       string(%d) "Elasticsearch\Client->index"
--
========DONE========
FAIL Elasticsearch userland functions are supported [tests/017-elastic-7-support.phpt] 
```

Reply:
Created PR upstream in ezimuel/ringphp#7

Reply:
Worth noting that `elasticsearch/elasticsearch:^8.0` is not affected, as it does not depend on `ezimuel/ringphp` - this only fails for `elasticsearch/elasticsearch:^7.0`.

Reply:
Created issue upstream in elastic/elasticsearch-php#1256

Reply:
This is now resolved with the [latest ezimuel/ringphp release](https://github.com/ezimuel/ringphp/releases/tag/1.2.2)